common/src/main/java/com/codeforcommunity/email/EmailOperations.java

```diff
@@ -159,38 +188,44 @@ public void sendEmailToOneRecipient(
     if (!shouldSendEmails) {
       return;
     }
-
     logger.info(String.format("Sending email with subject `%s`", subject));
+    Email email = buildEmailSingleRecipient(sendToName, sendToEmail, subject, emailBody, new ArrayList<>());
+    this.sendEmail(email, subject);
+  }
 
+  private Email buildEmailMultipleRecipient(
+          HashSet<String> sendToEmails, String subject, String emailBody, List<AttachmentResource> attachments) {
     Email email =
-        EmailBuilder.startingBlank()
-            .from(senderName, sendEmail)
-            .to(sendToName, sendToEmail)
-            .withSubject(subject)
-            .withHTMLText(emailBody)
-            .buildEmail();
+            EmailBuilder.startingBlank()
+                    .from(senderName, sendEmail)
+                    .bccMultiple(sendToEmails.toArray(new String[0]))
+                    .withSubject(subject)
+                    .withHTMLText(emailBody)
+                    .withAttachments(attachments)
+                    .buildEmail();
+
+    return email;
+  }
 
+  /** Send an email with the given subject and body to the users with the given email addresses with attachments. */
+  public void sendEmailToMultipleRecipients(
+      HashSet<String> sendToEmails, String subject, String emailBody, List<AttachmentResource> attachments) {
+    if (!shouldSendEmails) {
+      return;
+    }
+    logger.info(String.format("Sending emails with subject `%s`", subject));
+    Email email = buildEmailMultipleRecipient(sendToEmails, subject, emailBody, attachments);
```

Comment:
⛏️ Same idea as the above comment - is there something stopping us from just using method overloading here?